src/modules/memory.cpp

```diff
@@ -106,6 +112,10 @@ namespace modules {
       m_label->replace_token("%mb_total%", string_util::filesize_mb(kb_total, 0, m_bar.locale));
       m_label->replace_token("%percentage_used%", to_string(m_perc_memused));
       m_label->replace_token("%percentage_free%", to_string(m_perc_memfree));
+      m_label->replace_token("%percentage_available%", to_string(m_perc_memavail));
+      m_label->replace_token("%percentage_available_used%", to_string(m_perc_memavailused));
+      m_label->replace_token("%percentage_actualfree%", to_string(m_perc_memactualfree));
```

Comment:
I don't really like the name of these labels, but I don't have a better idea for now :/